src/src/WebServer/HTML_wrappers.cpp

```diff
@@ -83,6 +83,12 @@ void html_TD() {
   addHtml(F("<TD>"));
 }
 
+void html_TD(const String& style) {
```

Comment:
Maybe this should be called `html_TD_style()`?
As it only allows to set the style, not other arguments for the `TD` element.

Reply:
Or maybe it should be `void html_TD(const String& attribute, const String& value)`, so it can also be used to set f.e. the `colspan` attribute.